src/plugins/data/common/search/search_source/normalize_sort_request.test.ts

```diff
@@ -153,25 +160,6 @@ describe('SearchSource#normalizeSortRequest', function () {
     ]);
   });
 
-  it('should use script based sorting only on sortable types', function () {
```

Comment:
this is the test I've referring to in my next comment which throws an exception, but if this is not intended, this test should be rewritten to expect an exception, or the change that's now causing this failure should be undone